controller/konnect/consts.go

```diff
@@ -0,0 +1,9 @@
+package konnect
+
+const (
+	// AnnotationPrefix is the prefix for Kong annotations.
+	AnnotationPrefix = "konghq.com"
```

Comment:
Just to avoid using this prefix for Konnect-specific annotations (in case we have any in the future). Currently this is located in `konnect` package which could suggest it's a Konnect-specific prefix. 
```suggestion
	KongAnnotationPrefix = "konghq.com"
```

Reply:
I've added this as part of https://github.com/Kong/kubernetes-configuration/pull/45. After merging that I'll remove the consts here.

WDYT?